datalake-dr-connector/src/main/java/com/sequenceiq/cloudbreak/datalakedr/config/DatalakeDrConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.sequenceiq.cloudbreak.datalakedr.config;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DatalakeDrConfig {
+
+    private static final int DEFAULT_DATALAKE_DR_PORT = 80;
+
+    @Value("${altus.datalakedr.endpoint:}")
```

Comment:
If we would like to connect one of the cp service then we need changes in dps-k8s repo

Reply:
I have a PR here is to add this config setting in the CSI repo: https://github.infra.cloudera.com/thunderhead/cloud-services-infra/pull/1741. Is that what you mean, or is the dps-k8s repo different?

Reply:
The CSI change alone isn't doing anything. If you want to wire up the 2 services you'll need to make changes in CB helm charts: https://github.infra.cloudera.com/thunderhead/dps-k8s/tree/master/cloudbreak/helm-charts/cloudbreak-umbrella/charts

Reply:
Thanks. I'll look into that now.

Reply:
I think the colon should be removed and a default value must be included in the application.yaml which points to local mock, because right now it won't start locally

Reply:
The colon is in all the altus service endpoint settings (`"${altus.ums.host:}", "${altus.idbmms.host:}", "${altus.audit.endpoint:}"`, etc.) so I don't want to remove it, but I've added the altus.datalakedr.endpoint setting pointing to the datalake's application.yml file.